core/pta/stats.c

```diff
@@ -11,6 +11,7 @@
 #include <string.h>
 #include <string_ext.h>
 #include <malloc.h>
+#include <kernel/tee_time.h>
```

Comment:
Shoudl follow the alphabetical order.
Bt the way, could you also the other `#include` directives?
```c
#include <compiler.h>
#include <kernel/pseudo_ta.h>
#include <kernel/tee_time.h>
#include <malloc.h>
#include <mm/tee_mm.h>
#include <mm/tee_pager.h>
#include <stdio.h>
#include <string.h>
#include <string_ext.h>
#include <trace.h>
```

Reply:
will do.